### PR TITLE
Add `-j` flag to `anvill.py`

### PR DESCRIPTION
**Pull request by tetsuo-cpp**

### CODE REVIEW

File: tool_run_scripts/anvill.py [hunk 1]
```diff
@@ -155,7 +155,6 @@ def make_tool_cmd(self):
             str(self.tmpout),
             "-stats_out",
             str(self.stats_file),
-            "-remove_next_pc_assignments",
```
Comment (tetsuo-cpp) [inline tool_run_scripts/anvill.py:158]: Not sure where this came from but the Anvill in `patchable-ir-main` doesn't recognise this.

File: tool_run_scripts/stats.py [hunk 2]
```diff
@@ -80,11 +80,10 @@ def print_fails(self, fail_count=5, output=None, verbose=True):
 
     def get_fail_count(self):
         success_runs = len(self.stats.get("output.success", []))
-        timed_out_runs = self.stats.get("program_timeouts", 0)
         program_runs = self.stats.get("program_runs", 0)
         ignored_outputs = len(self.stats.get("outputignore_success", []))
         ignored_outputs += len(self.stats.get("outputignore_fail", []))
-        return (program_runs - ignored_outputs) - (success_runs + timed_out_runs)
+        return program_runs - ignored_outputs - success_runs
```
Comment (tetsuo-cpp) [inline tool_run_scripts/stats.py:86]: This is another minor fix that I've included.

The `timed_out_runs` stat includes ignored tests so we can get a negative fail count here. Also timing out should be considered a failure if we're not ignoring the test (in my opinion).

Comment (2over12) [reply tool_run_scripts/stats.py:86]: hmm yeah this was a temprorary fix to try to get anvill CI passing with the timeouts we have but I agree in principle

**Review by 2over12 — APPROVED** [review]
LGTM

**Pull request merged**